src/pages/index.jsx

```diff
@@ -1,28 +1,39 @@
-import Head from "next/head";
 import React from "react";
+import Image from "next/image";
 import Button from "../atoms/button";
-import useGet from "../ions/hooks/fetch/get";
-import Layout from "../organisms/layout";
+import styled from "@emotion/styled";
 
-const Page = () => {
-	const { data, loading, error } = useGet("/api/hello");
+// image link:
+// https://images.unsplash.com/photo-1514306191717-452ec28c7814?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1169&q=80
+
+const StyledImage = styled(Image)`
+	border-radius: 100px;
+`;
 
+const A = ({ children, className }) => {
```

Comment:
`A` isn't a good naming convention for a `const`. Try to find a better name.

Reply:
Thank you, that is correct, it was an exercise, and will be fixed!